stories/cloudprogressbar.js

```diff
@@ -0,0 +1,45 @@
+//@flow
+import React from "react";
+import CloudProgressBar from "../src/lib/components/cloudprogressbar/CloudProgressBar.component";
+import { Wrapper, Title } from "./common";
+import { storiesOf } from "@storybook/react";
+
+storiesOf("CloudProgressBar", module).add("Default", () => {
+  return (
+    <Wrapper>
+      <Title>CloudProgressBar</Title>
+      <div style={{ width: "200px", margin: '20px 0 20px 0' }}>
+        <CloudProgressBar
+          percentage={30}
+          borderSize='0.5px'
+          cloudColor='yellow'
+          progressCloudColor='blue'
+        />
+      </div>
+      <div style={{ width: "300px", margin: '20px 0 20px 0' }}>
+        <CloudProgressBar
+          percentage={50}
+          borderSize='0.5px'
+          cloudColor='#ccc'
+          progressCloudColor='#000'
+        />
+      </div>
+      <div style={{ width: "400px", margin: '20px 0 20px 0' }}>
+        <CloudProgressBar
+          percentage={80}
+          borderSize='1px'
+          cloudColor='#3d383a'
+          progressCloudColor='#83d136'
+        />
+      </div>
+      <div style={{ width: "500px", margin: '20px 0 20px 0' }}>
+        <CloudProgressBar
+          percentage={70}
+          borderSize='0.5px'
+          cloudColor='#BDFFB0'
+          progressCloudColor='#A3FFCB'
```

Comment:
It's better to specify the `children` here
```
          <div
            style={{
              position: "absolute",
              top: "50%",
              left: "50%",
              transform: "translate(-50%,-50%)"
            }}
          >
            <Title>70%</Title>
            <SubTitle>of the quotas has been reached.</SubTitle>
          </div>
```

Don't forget to import the `SubTitle` so what the text can support light/dark mode ;)
`import { Wrapper, Title, SubTitle } from "./common";`

Please add children in all the clouds.